README.md

```diff
@@ -290,6 +290,10 @@ Field int `gorethink:"myName,omitempty"`
 // the field is skipped if empty.
 // Note the leading comma.
 Field int `gorethink:",omitempty"`
+// When the tag name includes an index expression
+// a compound field is created
+Field1 int `gorethink:"myName[0]"`
+Field2 int `gorethink:"myName[1]"`
```

Comment:
Would be good to copy the example from the CHANGELOG here since users will see the readme but not generally the changelog.